art/estimators/classification/blackbox.py

```diff
@@ -287,3 +306,91 @@ def loss(self, x: np.ndarray, y: np.ndarray, **kwargs) -> np.ndarray:
 
     def compute_loss(self, x: np.ndarray, y: np.ndarray, **kwargs) -> np.ndarray:
         raise NotImplementedError
+
+
+@total_ordering
+class FuzzyMappingWrapper:
```

Comment:
Can we save the word Wrapper?
```suggestion
class FuzzyMapping:
```